tests/test_client.py

```diff
@@ -57,10 +57,8 @@
 
 
 @pytest.fixture
-def event_loop():
-    loop = async_solipsism.EventLoop()
-    yield loop
-    loop.close()
+def event_loop_policy():
```

Comment:
Is this magically picked up?

Reply:
Yes, it's a fixture provided by pytest-asyncio>=0.23 for you to override.